common/lib/xmodule/xmodule/course_module.py

```diff
@@ -1244,7 +1244,7 @@ def get_discussion_blackout_datetimes(self):
                     raise ValueError
             return ret
         except (TypeError, ValueError):
-            log.exception(
+            log.info(
                 "Error parsing discussion_blackouts %s for course %s",
```

Comment:
Nah, mostly for json formed in a way that `get_discussion_blackout_datetimes` doesn't expect. I think we validate on the front end that we get valid json, but that method expects it to be in a certain form.